sound/soc/sof/mediatek/mt8186/mt8186.h

```diff
@@ -52,7 +52,8 @@ struct snd_sof_dev;
 #define ADSP_PRID			0x0
 #define ADSP_ALTVEC_C0			0x04
 #define ADSP_ALTVECSEL			0x0C
-#define ADSP_ALTVECSEL_C0		BIT(1)
+#define ADSP_ALTVECSEL_C0		BIT(0)
+#define ADSP_ALTVECSEL_C1		BIT(1)
```

Comment:
The 'C0' means the first core in the specification. In the near future, there's no plan for the 2nd core. I feel that it's better to change it to BIT(1) | BIT(0) to prevent confusion on the naming.